### DISCUSSION TITLE
Tabler design template/components

### DISCUSSION THREAD

**Discussion by DanieleRigo** [tabler/tabler#1194] [General]
Are Tabler design template/components to mockup interfaces available somewhere?  
either for Figma, Illustrator or similar. 

IMHO Design components would be very useful if an user is planning to start using Tabler and wants to experiment with the design.

**Comment by Nepo26** [comment]
Any follow up here? Looking around it seems like that it wasn't created.

**Comment by Nepo26** [comment]
I will look into creating one.
